amount_test.go

```diff
@@ -807,3 +809,40 @@ func TestAmount_Scan(t *testing.T) {
 		})
 	}
 }
+
+func TestAmount_RoundTo_is_concurrency_safe(t *testing.T) {
```

Comment:
Let's call this TestAmount_RoundToWithConcurrency to keep the number of underscores to a minimum :)